leshan.py

```diff
@@ -134,9 +221,10 @@ def main():
     parser.add_argument('-host', '--hostname', help='Leshan server hostname or ip', default='leshan')
     parser.add_argument('-port', '--port', help='Leshan server port', default='8080')
     parser.add_argument('-m', '--monitor', help='Monitor the update', action='store_true', default=False)
-    parser.add_argument('-d', '--device', help='Device tyepe filter', default=None)
+    parser.add_argument('-d', '--device', help='Device type filter', default=None)
+    parser.add_argument('-t', '--threads', help='Maximum threads', default=1)
```

Comment:
You can add type=int here to type check the inputs, instead of recasting the input below.